scripts/ui-injector.js

```diff
@@ -14,6 +14,15 @@ addOnLoadObserver(`${PROJECT_SECTION_SELECTOR}`,
     void injectSaveTemplate(projectInputsDiv);
 });
 
+// Inject duration shortcuts before before the original duration data-element
+addOnLoadObserver(`${PROJECT_SECTION_SELECTOR} > div:nth-of-type(8)`, 
+    (durationDataRow) => {
+    console.debug("Duration Data div found.");
+    void injectDurationShortcutsTemplate(durationDataRow);
+})
+.catch(error => console.info('Error while finding the Duration Data div container: ', error.message));
```

Comment:
Remove the catch, because `addOnLoadObserver` is not returning a promise.

```suggestion
});
```

Currently, it causes this error: `Uncaught TypeError: Cannot read properties of undefined (reading 'catch')`